src/UI/Art/SpriteArtist/UnitSpriteSet.java

```diff
@@ -241,4 +207,18 @@ public void drawUnitIcons(Graphics g, Commander[] COs, Unit u, int animIndex, in
       g.drawImage( captureIcon, iconX, iconY, iconW, iconH, null );
     }
   }
+
+  public static boolean isStateFlippable(AnimState state)
+  {
+    switch(state)
+    {
+      case MOVENORTH:
+      case MOVEEAST:
+      case MOVESOUTH:
+      case MOVEWEST:
+        return false;
```

Comment:
Why not east/west?

Reply:
See above where I mentioned it's simpler to just have all the directions than figure it out at runtime.
I suppose we could just standardize and hold one copy for east/west, and just *always* flip the other direction?

Reply:
Yeah, that could work. You have to do the check when you first read the file anyway, so you could just do the flip there (if needed) and simplify the logic everywhere else.

Reply:
Not having to have flipping code in the draw function would be cool, but we technically store sprites relative to faction/colors rather than CO instance or HQ, so we'd have to change that in order to do it properly.
I feel like faction/colors is also the *right* level to store them, too, as that means we don't have to keep multiple copies around for different armies with the same faction/colors over the life of the application.

Reply:
Hm, I guess I wasn't clear enough in how I said things. Let's try again, because I thought I was agreeing with you before, but your latest response implies otherwise.
Currently, before this PR, all sprites are stored on disk as right/east facing. We read/store it that way, and when drawing we flip it if needed to face left/west.
In this PR as it stands, it checks for east and west, reads either/both, and copies one to the other if one is missing. This means it independently stores two sets of sprites when one set (plus flipping on draw) would suffice (as in the era before this PR).
I'm not suggesting to remove flipping code from the draw function, just to avoid storing two sets of east/west sprites; the old/existing logic would suffice to draw flipped or not without needing the extra storage.
This would be easy to implement because you already have to know which way it's facing when you load it from disk; you could just flip west sprites to face east at load and assume everything is east-facing from there on (though you'd still have to handle north/south).

Or, in short:
> I suppose we could just standardize and hold one copy for east/west, and just always flip the other direction?

Right?

Reply:
> flip west sprites to face east at load and assume everything is east-facing from there on (though you'd still have to handle north/south).

Idle/tired is "must flip if applicable"
Move north/south is "maybe flip"
Move east is "never flip"
Move west is "always flip"

I feel like that's more complex, not simpler.

Reply:
All right, let's approach this a different way. You said:
> I suppose we could just standardize and hold one copy for east/west, and just always flip the other direction?

Clearly I have no idea what you mean by it, because every time I attempt to explain what it means you say I'm wrong. Pretend I'm three. What are you trying to say?

Reply:
You interpreted my statement correctly.
It doesn't make the logic everywhere else simpler, though.

Reply:
I suspect that the logic used was shaped by the available resources, and that it wouldn't be particularly more complex (move east/west can be the same anim state, and the `flip` parameter gets used), but I also don't think I care enough given the probably-trivial amount of RAM it would save.
Accepting either east/west but then only using east if it's there _is_ a bit odd, to be fair, but I'm not sure I see us making non-ambidextrous sprites (unless you do it just so I'm wrong here).

Reply:
> I suspect that the logic used was shaped by the available resources

Not particularly? I made the logic to accept either, then realized that I'd set up the images wrong and renamed them to face the right direction :facepalm: 
> it wouldn't be particularly more complex (move east/west can be the same anim state, and the `flip` parameter gets used)

IMO the `flip` parameter should always match whatever the CO says.
> Accepting either east/west but then only using east if it's there _is_ a bit odd, to be fair, but I'm not sure I see us making non-ambidextrous sprites (unless you do it just so I'm wrong here).

You know me too well.

Reply:
> IMO the flip parameter should always match whatever the CO says.

Ahh, that kinda hurts my head. I'd want to rename it to `coFacing` or something, so that it's clear it doesn't apply in all cases. Would it be better to have an overload that just doesn't take in that parameter instead of accepting and not using it?

Reply:
> Ahh, that kinda hurts my head

Huh. Why's that?
> I'd want to rename it to coFacing or something, so that it's clear it doesn't apply in all cases.

That's super cool with me. Still in the range of pithy, with a significant clarity advancement.
> Would it be better to have an overload that just doesn't take in that parameter instead of accepting and not using it?

My worry in that case that we'd use it for IDLE/TIRED, and get dumb results.
I guess we could like, `assert` that it's not being used for those cases, but that honestly feels even more questionable.
Maybe what we should do is just bring back `Commander` into the param list, and let that function fetch the facing for that commander in case it's needed? That should bring the invocations of that logic down to 1.